shell/platform/android/io/flutter/embedding/android/AndroidTouchProcessor.java

```diff
@@ -220,6 +224,28 @@ private void addPointerForIndex(
     }
 
     int pointerKind = getPointerDeviceTypeForToolType(event.getToolType(pointerIndex));
+    // We use this in lieu of using event.getRawX and event.getRawY as we wish to support
+    // earlier versions than API level 29.
+    float viewToScreenCoords[] = {event.getX(pointerIndex), event.getY(pointerIndex)};
+    transformMatrix.mapPoints(viewToScreenCoords);
+    long buttons;
+    if (pointerKind == PointerDeviceKind.MOUSE) {
+      buttons = event.getButtonState() & 0x1F;
+      if (buttons == 0
+          && event.getSource() == InputDevice.SOURCE_MOUSE
+          && pointerChange == PointerChange.DOWN) {
+        // Some implementations translate trackpad scrolling into a mouse down-move-up event
```

Comment:
can `_POINTER_BUTTON_PRIMARY` be removed?  Do we know why this was introduced in #11420?

Reply:
@dkwingsmt

Reply:
Yes, we should remove this. It has no more uses.